homestar-runtime/tests/utils.rs

```diff
@@ -58,7 +61,7 @@ pub(crate) fn stop_homestar() -> Result<()> {
     Ok(())
 }
 
-/// Stop the IPFS daemon.
+/// Stop the IPFS binary.
```

Comment:
we should probably force remove this lock. 

```Error: cannot acquire lock: creation of lock C:\Users\runneradmin\.ipfs\repo.lock failed: The process cannot access the file because it is being used by another process.```

Reply:
Coming in https://github.com/ipvm-wg/homestar/pull/359. Thanks! 🙂